scripts/drone/rgm.star

```diff
@@ -333,10 +336,67 @@ def rgm_main_pipeline():
         rgm_main(),
     ]
 
+def rgm_promotion_pipeline():
+    """Promotion build pipeline.
+
+    Returns:
+      Drone pipeline.
+    """
+    promotion_trigger = {
+        "event": ["promote"],
+        "target": "rgm",
+    }
+
+    env = {
+        "GO_VERSION": golang_version,
+        "ALPINE_BASE": images["alpine"],
+        "UBUNTU_BASE": images["ubuntu"],
+    }
+
+    # Expected promotion args:
+    # * GRAFANA_REF = commit hash, branch name, or tag name
+    # * ENTERPRISE_REF = commit hash, branch name, or tag name. If not building an enterprise artifact, then this can be
+    #   left empty.
+    # * ARTIFACTS = comma delimited list of artifacts (ex: "targz:grafana:linux/amd64,rpm:grafana:linux/amd64")
+    # * VERSION = version string of Grafana that is being built (ex: v10.0.0)
+    # * UPLOAD_TO = Google Cloud Storage URL to upload the built artifacts to. (ex: gs://some-bucket/path)
+    build_step = {
+        "name": "rgm-build",
+        "image": "grafana/grafana-build:main",
+        "pull": "always",
+        "commands": [
+            "dagger run --silent /src/grafana-build artifacts " +
```

Comment:
Is there a specific reason why you don't just change the working directory to /src/grafana-build for this?

Reply:
yeah. In Drone, the working directory is the shared volume. If I changed working directories to /src, then the artifacts that were created wouldn't be shared to the next step.

Reply:
Ah, right! Thanks 🙂